wp-content/themes/workingnyc/views/home.twig

```diff
@@ -45,8 +45,8 @@
     <div class="bg-image-grid hidden tablet:grid grid-cols-2 animated fadeIn">
       <div></div>
 
-      <div class="bg-image-grid__mask-scale-1 relative">
-        <img class="bg-image-grid__img" loading="lazy" style="object-position: {{ post.thumbnail.focal_point }};" src="{{ post.thumbnail.src('desktop') }}" alt="{{ post.thumbnail.alt }}"/>
+      <div class="relative">
+        <img class="bg-image-grid__img rounded-full" loading="lazy" style="object-position: {{ post.thumbnail.focal_point }};" src="{{ post.thumbnail.src('desktop') }}" alt="{{ post.thumbnail.alt }}"/>
```

Comment:
- [ ] The background image is getting cut off when I resize the browser. Please put it in the grid columns so that it doesn't get cut off
- [ ] The height and width of the image should get smaller as the browser size decreases
- [ ] The image should not be visible on mobile

<img width="1115" alt="image" src="https://github.com/CityOfNewYork/working-nyc/assets/8957837/ba78253f-f6b3-49bf-b62c-3cc971ece62d">

<img width="1228" alt="image" src="https://github.com/CityOfNewYork/working-nyc/assets/8957837/8be24a23-d62f-408c-96b9-7539a22dca7c">

<img width="521" alt="image" src="https://github.com/CityOfNewYork/working-nyc/assets/8957837/53c80822-9f18-4fd8-8842-5c330eec3dd8">